Core/Core/Assignments/AssignmentList/ViewModel/AssignmentListViewModel.swift

```diff
@@ -189,6 +194,12 @@ public class AssignmentListViewModel: ObservableObject {
     }
 
     private func filterAssignments(_ assignments: [Assignment]) -> [Assignment] {
+        // Filtering by grading period except if all is selected (nil)
```

Comment:
```suggestion
        // Filtering by grading period except if "All" is selected (nil)
```
to clarify (without needing to check elsewhere) that All is an option, not all of the options